<launch>
    <arg name="basket" default="blue" />
    <arg name="ip" default="/localhost:9020" />
    <group ns="main">
        <node pkg="lebot" type="state_machine" name="Logic" />
    </group>
    <group ns = "Movement">
        <node pkg="lebot" type="myControllerInput" name="ControllerToRos" />
        <node pkg="lebot" type="theWheel" name="RosToWheel"/>
    </group>
    <group ns="Visual">
        <node pkg="lebot" type="theBall" name="FindBall"/>
        <node pkg="lebot" type="theBasket" name="FindBasket"/>
    </group>
    <include file="$(find realsense2_camera)/launch/rs_camera.launch">
        <arg name="align_depth" value="true"/>
    </include>
</launch>
